### ISSUE TITLE
Where to store or how to use token and id after log in?

### ISSUE THREAD

**Issue by mordamax** [kahmali/meteor-restivus#191]
Hello, I've been playing a bit with authentication via HTTP and I'm pretty new in Meteor. Here's my problem;

I can register via POST /api/users, I get token and user id
I can login via POST /api/login, I get token and userId as well 
but Meteor.user() or Meteor.userId are still null. After page reload is obviously the same
My configs are from README

What I tried to do is maybe hack:

```
Accounts.callLoginMethod({
   methodArguments: [{resume: 'KYN2LF7pklMTyECXe7O5EB-oA3IjpH9CX06ScmwjTtK'}],
   _suppressLoggingIn: true,
  userCallback: function (error) {console.log(arguments)}
})
```
And I get updated Accounts\Meter.userId() and user() imidiatelly. Unfotunatelly this data disappears after page reload. I suppose it should be preset somewhere in localstorage, but I am not sure it's good to describe what and where to save after XHR to /api/login, I thought it should be done automatically from the server?

I feel like I miss some part of whole the correct process -- how to login via HTTP in a correct way?

Thank you in advance for the response!
Max

**Comment by kahmali** [comment]
It sounds like you're trying to setup login for a Meteor client. That's not what Restivus is ideal for. It uses REST over HTTP, which is great for connecting external apps, like native mobile apps, to your Meteor server. But from a Meteor client, you have the awesomeness of DDP over websockets, and you should definitely use that for handling Meteor app login. Check out the `accounts-password` or other related [accounts](https://www.meteor.com/accounts) packages. I believe that's what you're looking for. Otherwise, I'm misunderstanding you.

**Comment by kahmali** [comment]
So to explain, Restivus doesn't log you into a Meteor client. It logs you in on the server and provides a token that you can use to make subsequent authenticated requests to the server (for any endpoints where you set `authRequired: true` ). The Meteor [accounts](https://www.meteor.com/accounts) package logs you in on the server and then stores and manages the auth token for you on the client. Restivus is a server-only package and doesn't touch the client.

**Comment by mordamax** [comment]
@kahmali thank you very much for heads up! So to create "REST API with bearer authentication" - what should I do and is there any examples of this? Or how the app process will look like?

**Comment by kahmali** [comment]
Yes, there was a very detailed discussion of this [here](https://github.com/kahmali/meteor-restivus/issues/129). I can't always respond quickly, so try searching the github issues (and stack overflow if that doesn't work). This one would have been easy to find with a search for "bearer authentication." Just a heads up for next time.
